polkadot/runtime/parachains/src/disputes.rs

```diff
@@ -853,6 +852,14 @@ impl StatementSetFilter {
 }
 
 impl<T: Config> Pallet<T> {
+	/// Whether the chain is frozen. Starts as `None`. When this is `Some`,
+	/// the chain will not accept any new parachain blocks for backing or inclusion,
+	/// and its value indicates the last valid block number in the chain.
+	/// It can only be set back to `None` by governance intervention.
+	pub fn last_valid_block() -> Option<BlockNumberFor<T>> {
+		Frozen::<T>::get()
+	}
+
```

Comment:
Why's this needed?

Reply:
It is not, sorry. I'll remove it and set the `StorageValue` as `pub` as it was done for the other items